bridge-ext/src/test/java/com/liferay/faces/bridge/ext/renderkit/html_basic/internal/UICommandMockImpl.java

```diff
@@ -25,21 +25,31 @@
 public class UICommandMockImpl extends UICommand {
 
 	// Private Data Members
-	private String componentFamily;
-	private Map<String, Object> passThroughAttrs;
-	private String rendererType;
+	private final String componentFamily;
+	private final Map<String, Object> passThroughAttrs;
```

Comment:
@stiemannkj1 Might be a little silly for a mock, but I think we decided that `final Map` would always be unmodifiable.